examples/nextjs-auth/package-lock.json

```diff
@@ -1,12939 +0,0 @@
-{
```

Comment:
we should check in this lock file - we want to make sure that the example "just works" for developers using the example